charts/base-cluster/templates/weave/gitops-dashboard.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.weaveworks.enabled }}
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  annotations:
+    metadata.weave.works/description: This is the Weave GitOps Dashboard.  It provides
+      a simple way to get insights into your GitOps workloads.
+  name: gitops-dashboard
+  namespace: flux-system
+spec:
+  chart:
+    spec: {{- include "base-cluster.helm.chartSpec" (dict "repo" "weaveworks" "chart" "weave-gitops" "context" $) | nindent 6 }}
+  interval: 5m
+  values:
+    replicaCount: 1
+    {{- if and .Values.weaveworks.ingress .Values.weaveworks.ingress.enabled }}
+    ingress:
+      enabled: true
+      host: {{ include "base-cluster.weaveworks.host" $ }}
+    {{- end }}
+    metrics:
+      enabled: {{ .Values.monitoring.prometheus.enabled }}
+    {{- with .Values.global.authentication }}
+    {{- if and .config.clientId $.Values.weaveworks.ingress $.Values.weaveworks.ingress.host }}
```

Comment:
@cwrau not sure if the redirectURL is actual mandatory for the OIDC :thinking:. However, due to ci issues I disabled the oidc-creation if the ingress is not configured for the gitops-dashboard. We can also drop the redirectURL and as the result the condition as well, if we do not need the redirectURL at all